liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/PluginConfigSupport.java

```diff
@@ -0,0 +1,192 @@
+    private String appsDirectory = "dropins";
+    
+    /**
+     * Strip version.
+     */
+    @Parameter(property = "stripVersion", defaultValue = "false", readonly = true)
+    private boolean stripVersion = false;
+    
+    /**
+     * Loose application. 
+     */
+    @Parameter(property = "looseApplication", defaultValue = "false", readonly = true)
+    private boolean looseApplication;
+    
+    @Component
+    private BuildContext buildContext;
+    
+    protected final String PLUGIN_CONFIG_XML = "liberty-plugin-config.xml";
+    
+    @Override
+    protected void installServerAssembly() throws Exception {
+        File f = exportParametersToXml();
+        super.installServerAssembly();
+        this.buildContext.refresh(f);
+        this.buildContext.refresh(installDirectory);
+    }
+    
+    /*
+     * Export plugin configuration parameters to target/liberty-plugin-config.xml
+     */
+    protected File exportParametersToXml() throws Exception {
+        PluginConfigXmlDocument configDocument = PluginConfigXmlDocument.newInstance("liberty-plugin-config");
+        
+        @SuppressWarnings("unchecked")
+        List<Profile> profiles = project.getActiveProfiles();
+        configDocument.createActiveBuildProfilesElement("activeBuildProfiles", profiles);
+        
+        configDocument.createElement("installDirectory", installDirectory);
+        configDocument.createElement("serverDirectory", serverDirectory);
+        configDocument.createElement("userDirectory", userDirectory);
+        configDocument.createElement("serverOutputDirectory", new File(outputDirectory, serverName));
+        configDocument.createElement("serverName", serverName);
+        configDocument.createElement("configDirectory", configDirectory);
+        
+        if (getFileFromConfigDirectory("server.xml", configFile) != null) {
+            configDocument.createElement("configFile", getFileFromConfigDirectory("server.xml", configFile));
+        }
+        if (getFileFromConfigDirectory("bootstrap.properties", bootstrapPropertiesFile) != null) {
+            configDocument.createElement("bootstrapPropertiesFile", getFileFromConfigDirectory("bootstrap.properties", bootstrapPropertiesFile));
+        } else {
+            configDocument.createElement("bootstrapProperties", bootstrapProperties);
+        }
+        if (getFileFromConfigDirectory("jvm.option", jvmOptionsFile) != null) {
+            configDocument.createElement("jvmOptionsFile", getFileFromConfigDirectory("jvm.option", jvmOptionsFile));
+        } else {
+            configDocument.createElement("jvmOptions", jvmOptions);
+        }
+        if (getFileFromConfigDirectory("server.env", serverEnv) != null) {
+            configDocument.createElement("serverEnv", getFileFromConfigDirectory("server.env", serverEnv));
+        }
+        
+        configDocument.createElement("appsDirectory", appsDirectory);
+        configDocument.createElement("looseApplication", looseApplication);
```

Comment:
What I was saying in that other review requested you to add "looseApplication" as you did, but you need to leave in the line: 
configDocument.createElement("looseConfig", looseApplication); 
for now until WDT changes it's code and has another beta update.  Otherwise the current beta release will fail.

Also add a comment why you need both lines for now and that the "looseConfig" element should be removed later.